spec/install/gemfile/git_spec.rb

```diff
@@ -1010,13 +1009,11 @@
       install_gemfile <<-G
         gem "foo", :git => "file://#{lib_path("foo-1.0")}", :ref => "#{revision}"
       G
-      bundle "install"
```

Comment:
these `bundle install`s probably shouldn't be removed?

Reply:
Aren't they redundant given that `install_gemfile` already runs that?

Reply:
👍